tendrl/monitoring_integration/alert/handlers/cluster/gluster_handler/brick_utilization_handler.py

```diff
@@ -19,9 +19,8 @@ class BrickHandler(AlertHandler):
 
     def __init__(self):
         AlertHandler.__init__(self)
-        self.template = "tendrl.clusters.{integration_id}.nodes."\
-            "{host_name}.bricks.{brick_path}.utilization."\
-            "percent-percent_bytes"
+        self.template = "tendrl[.]name[.]{integration_id}[.]nodes[.]"\
+            "{host_name}[.]bricks[.]{brick_path}[.]"
```

Comment:
In old regular expression template, we always have to give a word at the end. For example in this template "percent-percent_bytes" is given. But this template will do exact match only, no need any extra word at the end.

So whenever metric changes after brick_path we don't need to change the template.